openmetadata-ui/src/main/resources/ui/src/components/ProfileCard/ProfileSectionUserDetailsCard.component.tsx

```diff
@@ -114,22 +116,26 @@ const ProfileSectionUserDetailsCard = ({
   const userEmailRender = useMemo(
     () =>
       !isMaskedEmail(userData.email) && (
-        <>
-          <Typography.Paragraph
-            className="m-b-0 profile-details-email"
-            data-testid="user-email-value">
-            {userData.email}
-          </Typography.Paragraph>
-        </>
+        <Typography.Paragraph
+          className="m-b-0 profile-details-email"
+          data-testid="user-email-value">
+          {userData.email}
+        </Typography.Paragraph>
       ),
     [userData.email]
   );
 
+  const handleRestore = async () => {
+    setIsRestoring(true);
+    handleRestoreUser();
```

Comment:
Is this an async function if yes, let's show the loader when in process!

Reply:
addressed